### ISSUE TITLE
When the training interruption resumes, the visualization of tensorboard seems to have a bug

### ISSUE THREAD

**Issue by YoRha19990213** [PDillis/stylegan3-fun#31]
The previous training ended at 140 rounds, I used the parameter "--resume-kimg=140" to continue this training, training to 260 rounds, but I found that the two tensorboard output log files did not lose equal when training to 140 rounds, what is the reason.
Or should I use "--resume-kimg=264" instead of 260 for my next follow-up training?

**This is the parameter I used for training**
![QQ截图20230322214921](https://user-images.githubusercontent.com/83154603/226925069-a2388296-9f12-4001-8dab-01069bda7515.png)

**This is the display panel of tesnorboard**
![QQ截图20230322215006](https://user-images.githubusercontent.com/83154603/226925364-69e3407e-3e1d-4a1f-ae4b-06336f85e3a2.png)

**Comment by PDillis** [comment]
As your previous snapshot where you're resuming training from is at `260kimg`, then you should use `--resume-kimg=260`. The fact that the values don't start from the exact same spot could be from a different `augment` strength at those points, assuming you were using ADA.

**Comment by YoRha19990213** [comment]
![QQ截图20230324192918](https://user-images.githubusercontent.com/83154603/227511392-ce623d87-a67f-49be-a283-dffe43092eb2.png)

Thank you for your reply. I did find that ADA was used during training, but ADA only affects the changes in the learning rate. When I continued training using the weight file from the previous checkpoint, the initial loss changed as well（For example, this breakpoint of kimg=260...）. Is this also due to the influence of ADA?
![QQ截图20230324195755](https://user-images.githubusercontent.com/83154603/227516099-681b1821-8b99-4591-80ac-e59e84bc6483.png)
![QQ截图20230324195810](https://user-images.githubusercontent.com/83154603/227516135-4ff6fe3c-56a7-4aea-b6be-01b47bb85784.png)

At the same time, I also found that the augmentation changes from 0 each time during training. Is there any parameter that can specify the value of augmentation when continuing training?
